### ISSUE TITLE
Bubbleshowcase Screenshot 

### ISSUE THREAD

**Issue by invisibleBully** [ECLaboratorio/BubbleShowCase-iOS#6]
This weird behavior happens sometimes can't seem to put a finger on it..
![Screenshot 2020-05-04 at 11 53 43 AM](https://user-images.githubusercontent.com/12860615/80963377-5675ed00-8dfe-11ea-8df5-386d6c4c97ef.png)

**Comment by PrashantTiwari123** [comment]
This weird behavior happens sometimes can't seem to put a finger on it.

![Screenshot 2020-05-18 at 11 46 37 PM](https://user-images.githubusercontent.com/32719759/82247286-c7bbb100-9963-11ea-944a-8f9098b34197.png)

**Comment by invisibleBully** [comment]
@PrashantTiwari123 you're experiencing this too I see

**Comment by PrashantTiwari123** [comment]
Yes App crashed and not able to find out this at my end. getting this
error.

partial apply forwarder for closure #2 () -> () in
BubbleShowCase.BubbleShowCase.(takeScreenshot in
_6C61E71B2EB682DC290B23F96E92C494)() -> () BubbleShowCase.swift:1009

Could you please help me out with this error.  Because this crash is
happened  many of the users.

Regards,
Prashant Tiwari

**Comment by lohithkorp** [comment]
I have the same problem too... anyone seems to figure out the cause please?
I need some help.

**Comment by iamkenjobs** [comment]
Hi @invisibleBully @lohithkorp I fixed this by applying a nil checker making sure that the self.target is not nil, before executing the code. I hope this helps

            if let parent: UIView = self.target.superview {
                if let screenshot = parent.resizableSnapshotView(from: self.target.frame, afterScreenUpdates: false, withCapInsets: UIEdgeInsets.zero) {
                    screenshot.backgroundColor = self.sreenshotContainerBackground
                    screenshot.translatesAutoresizingMaskIntoConstraints = false
                    screenshot.isUserInteractionEnabled = false
                    screenshotShadow.addSubview(screenshot)
                    
                    let centerXImage = screenshot.centerXAnchor.constraint(equalTo: screenshotShadow.centerXAnchor)
                    let centerYImage = screenshot.centerYAnchor.constraint(equalTo: screenshotShadow.centerYAnchor)
                    let widthImage = screenshot.widthAnchor.constraint(equalToConstant: targetFrame.width)
                    let heightImage = screenshot.heightAnchor.constraint(equalToConstant: targetFrame.height)
                    screenshotShadow.addConstraints([centerXImage, centerYImage, heightImage, widthImage])
                }

            }

**Comment by Sukarma33** [comment]
@iamkenjobs @fermoya it doesn't work, I tried to add the effect of changing tabbar image when showing case, but it always force close in the code "let parent: UIView = self.target.superview ?? self.target", I don't know why
